src/main/routes.ts

```diff
@@ -25,6 +25,8 @@ export default function(app: Application): void {
   app.get('/services/:service/:serviceArea/search-by-postcode', app.locals.container.cradle.servicePostcodeSearchController.get);
   app.get('/services/:service/:serviceArea/courts/near', app.locals.container.cradle.servicePostcodeResultsController.get);
   app.get('/not-found', app.locals.container.cradle.notFoundPageController.get);
+  app.get('/v2/proxy/search/postcode=:postcode&serviceArea=:serviceArea', app.locals.container.cradle.proxyController.getCourtsByPostcodeServiceArea);
```

Comment:
can we have the route like  '/v2/proxy/search/postcode/:postcode/serviceArea/:serviceArea' would that work?